project/Dependencies.scala

```diff
@@ -71,24 +71,26 @@ object Dependencies {
   private val owlApiV = "5.1.16"
   private val paradiseV = "2.1.1"
   private val pegdownV = "1.6.0"
-  private val postgresV = "42.2.5"
+  // For org.postgresql:postgresql 42.2.6 - 42.2.14:
+  // java.lang.NoSuchFieldException: m_mesgParts in KeyValueSpec "fail if one of the inserts fails"
+  private val postgresV = "42.2.5" // scala-steward:off
```

Comment:
ToL: I suspect this is an issue with the Cromwell test that should be filed in Jira and investigated at a later date.

Reply:
Created as https://broadworkbench.atlassian.net/browse/BA-6557